### DISCUSSION TITLE
1.21.8 to 1.22

### DISCUSSION THREAD

**Discussion by Raz0rLegend** [Vdauphin/HeartsAndMinds#1213] [Question and answer] [answered]
Hello Vdauphin,
How will the transfer go from 1.21.8 to 1.22 with the BTC_veh module which will only exist with game logic?
Does the save on 1.21.8 will work anymore ?

Thanks in adavance !

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> How will the transfer go from 1.21.8 to 1.22 with the BTC_veh module which will only exist with game logic?

I don't understand. 

If you want to migrate from 1.21 to 1.22:
- You need to take the 1.22 pbo and build your mission on top of it (even if you have a 1.21 pbo already setup). You must not use any file from your previous 1.21 pbo because a lot of stuff have changed.
- If you have a save, your base must fit the old base
- Then, if everything is setup correctly in your 1.22 pbo, the mission will do basically what is doing at each loading, remove all vehicles from the mission.sqm: https://github.com/Vdauphin/HeartsAndMinds/blob/6992973d5ae35dfb0c95bd0ae59ac7ec074217dc/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/db/load_old.sqf#L126-L127
And then load vehicles saved inside the database: https://github.com/Vdauphin/HeartsAndMinds/blob/6992973d5ae35dfb0c95bd0ae59ac7ec074217dc/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/db/load_old.sqf#L209-L267
- If the new and old base fit each other, you will see no vehicles and objects clipping with objects from the new base.


> Does the save on 1.21.8 will work anymore ?

Yes, I provide database compatibility between hotfix and major release

Cheers

**Comment by Raz0rLegend** [comment]
So if I understand well. If inside the editor I let all my btc_veh and I trigger them to the new way to put the vehicles to the mission. It should work with the 1.21.8 save on the 1.22 ?

**Reply by Vdauphin** [reply]
Exactly
